Updates Readme file (#123)
microsoftgraph/microsoft-graph-devx-content: README.md

Follow the instructions specified, then run this command again:<br/>
`git push origin {your-branch-name}`

## Testing of Sample Queries

> Once you've added/updated the sample queries, you can test them out by going to [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer), and appending this to the Graph Explorer url.

Reviewer: ```suggestion
Once you've added/updated the sample queries, you can test them out by going to [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer), and appending this to the end of the Graph Explorer url.
```

> `?devx-api=https://graphexplorerapi.azurewebsites.net&org=microsoftgraph&branchName=dev`

Reviewer: The `org` and `branch` needs to be specific to where the samples were added in. Could be this repo in a diff. branch or a forked repo of this repo.

> `?devx-api=https://graphexplorerapi.azurewebsites.net&org=microsoftgraph&branchName=dev`

Reviewer: ```suggestion
`?devx-api=https://graphexplorerapi.azurewebsites.net&org={microsoftgraph or your GitHub username}&branchName={your branch name}`
```

> This overrides the default DevX API url that fetches from blob storage and instead, fetches the files from your branch by adding two optional query params to the url i.e **org** and **branchName** to represent the path of the file.

Reviewer: I think we can abstract out the technical details and just highlight that this would fetch the permissions/samples from the specified repo or branch.

#### Full url/path format:
```

> https://developer.microsoft.com/en-us/graph/graph-explorer?devx-api=https://graphexplorerapi.azurewebsites.net&org=microsoftgraph&branchName=dev

Reviewer: ```suggestion
https://developer.microsoft.com/en-us/graph/graph-explorer?devx-api=https://graphexplorerapi.azurewebsites.net&org={org}&branchName={branchName}
```

```
#### NB: 

> i. Replace *branchName* with the name of your current branch.

Reviewer: ```suggestion
i. Replace *{branchName}* with the name of your current branch.
```

> ii. In the case of a forked repo, replace the *org* parameter with your Github username.

Reviewer: ```suggestion
ii. In the case of a forked repo, replace the *{org}* parameter with your Github username. If your branch is on this repo, replace *{org}* with microsoftgraph
```

#### Example:

```
https://developer.microsoft.com/en-us/graph/graph-explorer?devx-api=https://graphexplorerapi.azurewebsites.net&org=MeganBowen&branchName=mk/update-sample
```

> The samples will be populated on Graph Explorer, you'll see a 200 status code and you can do further testing.

Reviewer: Where will this status code be displayed?

Author: Actually I've confirmed its not, I was using the network tab on developer tools to verify status code.
Good catch.
